Makefile.plugins.mk

```diff
@@ -6,20 +6,21 @@
 
 # NOTE: NAME, FILES_DSP and FILES_UI must have been defined before including this file!
 
+ROOT_DIR ?= ../..
```

Comment:
This seems completely wrong.
`?=` means environment variables can override this value, this is not what we want at all...

Reply:
No problem, this was only a very minor enhancement to simplify the usage of `Makefile.plugins.mk` for plugin specific makefiles in another directory structure. To prevent environment variables from overriding this value we could use `ROOT_DIR = ../..` instead and the plugin specific Makefile has to use `override ROOT_DIR = ....`. 

Without this enhancement the plugin specific Makefile has to use 
```
override DPF_PATH = ....
override TARGET_DIR = .....
override BUILD_DIR  = ....
```
IMHO that's also OK so the benefit of introducing `ROOT_DIR` can be seen as negligible.